### DISCUSSION TITLE
[New Feature] Regenerating/respawning map assets

### DISCUSSION THREAD

**Discussion by CCTiberius** [Vinifera-Developers/Vinifera#1504] [Feature Requests]
### Description:
The primary example I am thinking of is the ability for a veinhole monster to respawn/reappear, but perhaps there could be other applications such as destructible tib spreaders that regenerate as well, or flora that regrows.

### Possible Implementation:
A `Regrows=yes/no` key. It would be nice if users could actually randomize the time it takes, maybe setting it to happen somewhere between an X and Y amount of ticks of their choosing.

**Comment by Rampastring** [comment]
I wonder if a a trigger action for spawning objects (including buildings and terrain objects) could achieve this in a more flexible way? Having both could an option though ofc.

**Comment by Crimsonum** [comment]
This would also be a great feature for tech buildings, however it's implemented.

**Comment by CCTiberius** [comment]
@Rampastring I think it is best to avoid implementing via triggers whenever possible, having to do things on a per-map basis is annoying and besides I have the impression that simple INI coding is a far, far more common skill than even rudimentary triggerwork.
